__tests__/contact/contact.query.test.js

```diff
@@ -4,20 +4,22 @@ const { CONTACT_NOT_FOUND } = require('../../error-messages/contact.messages');
 const { newContact, notExistContactId } = require('./contact.variables');
 const { setupApp } = require('../helper-functions');
 jest.mock('../../modules/upload/upload.service');
-let a = 1;
+jest.mock('../../modules/contact/contact.utils.js');
+
 let contactsId = '';
 let operations;
 
 describe('Contacts queries', () => {
-  beforeAll(async () => {
+  beforeAll(async done => {
```

Comment:
Make sense to move it to global scope so that can work with all tests in the file

Reply:
`done` is an argument that we pass to arrow function.

Reply:
fixed